indy_node/test/request_handlers/test_context_handler.py

```diff
@@ -0,0 +1,267 @@
+
+import pytest
+
+from indy_common.authorize.auth_constraints import AuthConstraintForbidden
+from plenum.common.exceptions import UnauthorizedClientRequest
+
+from indy_common.req_utils import get_write_context_name, get_write_context_version
+from plenum.common.txn_util import get_request_data
+
+from common.exceptions import LogicError
+from indy_common.constants import CONTEXT_TYPE
+
+from indy_node.server.request_handlers.domain_req_handlers.context_handler import validate_data, validate_context, \
+    ContextHandler, validate_meta
+from plenum.common.request import Request
+from indy_node.test.context.helper import W3C_BASE_CONTEXT, W3C_EXAMPLE_V1_CONTEXT
+from plenum.server.request_handlers.utils import encode_state_value
+
+
+def test_validate_meta_fail_on_empty():
+    with pytest.raises(KeyError) as e:
+        validate_meta({})
```

Comment:
I think it would be better to call `ch.static_validation` in this and the following tests to make sure that static validation really throws expected errors. We are testing ContextHandler's API here, and `static_validation` is one of the API methods.

Reply:
fixed in [51fd316](https://github.com/hyperledger/indy-node/pull/1430/commits/51fd316b9346906f925fe1cd0f08780983f9387f)